contrib/node/src/python/pants/contrib/node/register.py

```diff
@@ -37,12 +39,19 @@ def build_file_aliases():
 
 
 def register_goals():
+  # Register descriptions.
```

Comment:
Another way to do this is to put a docstring on the task itself. The first line of your docstring is used as a description.